php/git.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+class Git {
```

Comment:
Wonder if we should just namespace this from day 1?

Reply:
You mean instead of using a class, right? I'm fine either way - this isn't really a class anyway.

Reply:
I was meaning as well as.

It probably is a little overkill having a class for one function, sure, but at some point we can rename it to something like Utils when more things are being created/refactored.

Not like we really need to worry about "back compat" etc.